Document/0x05b-Basic-Security_Testing.md

```diff
@@ -59,12 +59,86 @@ The first step in rooting a mobile is to unlock its boot loader. The procedure d
+
+1. In your Android device go to "Settings" menu.
+<img src="Images/Chapters/0x05b/android-settings.png"/>
+
+2. Go to "Network & Internet".
+<img src="Images/Chapters/0x05b/android-wifi-settings.png"/>
+
+3. Switch the "Wi-Fi" button on, and tap the "Wi-Fi".
+<img src="Images/Chapters/0x05b/android-wifi-on.png"/>
+
+4. You should be presented with a list of available networks. Find your network and connect to it.
+<img src="Images/Chapters/0x05b/mstg-network.png"/>
+
+5. Once you are connected, hold down on the network name to bring up the context menu. Tap "Modify network".
+<img src="Images/Chapters/0x05b/android-modify-network.png"/>
+
+6. Tap on "Advanced options" and then on "None" in "Proxy" section.
+<img src="Images/Chapters/0x05b/android-advanced-options.png"/>
+
+7. Choose "Manual".
+<img src="Images/Chapters/0x05b/android-proxy-manual.png"/>
+
+8. Specify the IP address of your computer where the proxy is listening into the "Proxy hostname". Enter the port number which you specified in the “Proxy Listeners” section earlier. Tap "Save".
+<img src="Images/Chapters/0x05b/android-proxy-hostname-port.png"/>
+
+9. Now you should be able to handle HTTP connection. To test it go to your Android browser and visit any HTTP web page.
+
+10. You should see the intercepted request in your proxy.
+<img src="Images/Chapters/0x05b/burp-intercepted-request.png"/>
+
+
+> Note that you are able to intercept only HTTP connections. To intercept also HTTPS you have to install Burp's CA certificate on your Android device. The below section explains how to do it.
+
+###### Installing CA certificate on Android device
+
+The CA certificate of the interception proxy need to be added to the trusted certificates in the certificate storage of the Android device. Due to different versions of Android and modifications of Android OEMs to the settings menu, the location of the menu to store the CA certificate might differ. The below mentioned example is based on Android 8.0 on Nexus 5x device.
+
+1. On your computer with running proxy, visit the following address: http://burp. In the top-right corner, right click on "CA Certificate" and rename the downloaded file with ".cer" extension, e.g. "cacert.cer".
+<img src="Images/Chapters/0x05b/burp-ca-certificate.png"/>
+
+2. Move the certificate file to your mobile device. You can do this in few ways by:
+	- sending it in a mail as an attachment and then receive it on a mobile device using mail client.
+	- sending it via USB cable and program [Android File Transfer](https://www.android.com/filetransfer/) or [Android Debug Bridge](https://developer.android.com/studio/command-line/adb.html).
+	- sending it via Bluetooth.
+	- sending it via web server.
+>This guide shows a method of accessing the "cacert.cer" file as a mail attachment.
+3. In your mail client open message you have just sent and download the "cacer.cer" file to your mobile device.
+<img src="Images/Chapters/0x05b/mail-cert-download.png"/>
```

Comment:
Instead of these 3 steps, you could just access http://192.168.1.6:8081/ on your mobile device to download the CA cert there.

Reply:
Not sure why, but this option doesn't work on my device :/ I can download a "cacert.der" file and rename it to "cacert.cer", but after that I get toast message "Can't open file". Do you have @Sjord the same issue?

Reply:
Look on attached screenshot (cert.cer file is downloaded from mail client and cacert.cer is a file downloaded from a mobile browser). They have the same size but when opening file cacert.cer I'm getting an error and opening cert.cer installs certificate.
<img width="304" alt="screen shot 2017-09-29 at 17 23 20" src="https://user-images.githubusercontent.com/17786325/31024002-f0996332-a53d-11e7-9cd7-3d0ca3e503a2.png">

Reply:
Yep, you need to change the file extension to .cer.